app/frontend/components/shared/base/inputs/user-input.tsx

```diff
@@ -111,11 +111,20 @@ const IInviteResultTag = ({ bg, icon, text, ...rest }: IInviteResultTagProps) =>
   const color = (bg as string).replace(/Light/g, "")
 
   return (
-    <Tag border="1px solid" borderColor={color} mb={2} noOfLines={1} bg={bg} {...rest}>
-      <HStack color={color}>
-        {icon}
-        <Text>{text}</Text>
-      </HStack>
+    <Tag
+      border="1px solid"
+      borderColor={color}
+      mb={2}
+      noOfLines={1}
+      bg={bg}
+      color={color}
+      {...rest}
```

Comment:
`{...rest}` should always be the final prop. That way the parent component can override any of the base props. Right now, it is not possible for the parent to override anything below the ...rest

Reply:
woops good catch